Automattic/wp-calypso
# Added email illustration for the onboarding checklist

#### Changes proposed in this Pull Request

This PR adds a email illustration for the email tasks in the onboarding checklist.

#### Testing instructions

1. Create a site that is eligible for G Suite (has a .blog domain).
2. Go to `checklist/<site_slug>` and complete every task except for the email task. Tip: you can click the little empty circle next to a task to "dismiss" many of the tasks.
3. Head to the `stats` page and verify that the onboarding checklist banner that shows displays the illustration.
4. Look at the code, and verify that the other forms of the task have an identical value for `bannerImageSrc` (or construct actual sites to cover those use cases as well if you prefer).
